infrastructure/k8s/services/auth.yaml

```diff
@@ -0,0 +1,36 @@
+project_name: "taskcluster-auth"
+secrets:
+  aws_access_key_id: ".Values.authAwsAccessKeyId"
+  aws_region: ".Values.tfAwsRegion"
+  aws_secret_access_key: ".Values.authAwsSecretAccessKey"
+  azure_account: ".Values.tfAzureAccount"
+  azure_accounts: ".Values.azureAccounts"
+  azure_account_key: ".Values.azureAccountKey"
+  azure_crypto_key: ".Values.authAzureCryptoKey"
+  azure_signing_key: ".Values.authAzureSigningKey"
+  force_ssl: "false"
+  lock_roles: "false"
+  node_env: "production"
+  owner_email: "cloud-ops@mozilla.com"
```

Comment:
We can drop this variable -- it was only used as an indicator to ourselves in Heroku of who was in charge.  I'm not sure why it ended up in Terraform.